week-1-practise/level-1/10-function-output-3.js

```diff
@@ -1,6 +1,7 @@
 function concatenate(firstWord, secondWord, thirdWord) {
-    // Write the body of this function to concatenate three words together
+  // Write the body of this function to concatenate three words together
+   return firstWord.concat(secondWord, thirdWord);
```

Comment:
Ha, I actually had forgotten that there was a `concat` function on strings! You've taught me something 😄 

Is there another style of string concatenation that you could use?

Reply:
I could use just the plus sign to concatenate all the arguments, it's more efficient than using _concat method_ :smile:

Reply:
I think they are probably equally efficient in terms of how long it takes the computer to run them. But yes, it is slightly easier to type. Either way is good though!